src/management-system-v2/app/(dashboard)/[environmentId]/machine-config/[configId]/mach-config.tsx

```diff
@@ -259,6 +272,7 @@ const MachineConfigurations: React.FC<MachineDataViewProps> = ({
           background: 'none',
         }}
         items={items}
+        defaultActiveKey={machineIds.current}
```

Comment:
yes, that's totally right. The whole useMemo is still unaltered legacy code. But I think duplicates won't be a problem since the configs are always appended to an empty list.